src/audio_worklet.js

```diff
@@ -203,7 +173,7 @@ class BootstrapMessages extends AudioWorkletProcessor {
 #else
         var ptr = d['_wsc'];
```

Comment:
Nit (unrelated to this PR) but should this just use the `to64` helper?

Reply:
I wrote a whole wasm64 conversion waiting to be simplified, and it wasn't always a simple as to which of the helpers were needed:

https://github.com/cwoffenden/emscripten/blob/6b3f01eb7327c2ee7f8056a4849bdddf536fb5c1/src/audio_worklet.js#L278

Reply:
Resolving as unrelated.